Sources/_StringProcessing/CharacterClass.swift

```diff
@@ -338,8 +338,6 @@ extension AST.Atom {
     switch kind {
     case let .escaped(b): return b.characterClass
 
-    case .any: return .any
-
```

Comment:
Removing this means that the `.any` case will be handled using a consumer function, rather than the old `CharacterClass` type.